src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res = 0;
```

Comment:
**[OPTIONAL]**
We could wrap `numberOne` and `numberTwo` in big.js and store them in a new variable one time. So we can keep calling them in the various conditional block.